### PR TITLE
Update workspace for pipenv and python 3.8

### PR DESCRIPTION
**Pull request by thrile**

### CODE REVIEW

File: .vscode/cirkopt.code-workspace [hunk 1]
```diff
@@ -5,8 +5,37 @@
 		}
 	],
 	"settings": {
-		"python.linting.pylintEnabled": false,
-		"python.linting.flake8Enabled": true,
-		"python.linting.enabled": true
-	}
+		// Add *.pyc and __pycache__ to excluded files
```
Comment (jameswolfeoliver) [inline .vscode/cirkopt.code-workspace:8]: Is this file needed for vscoded development? If not, would be nice to remove from tracking if its just an autogenerated thing

**Review by jameswolfeoliver — APPROVED** [review]

File: .vscode/cirkopt.code-workspace (hunk 1 above)
Comment (thrile) [reply .vscode/cirkopt.code-workspace:8]: This isn't auto generated, I wrote it

It is basically a project file for VS Code that tells it what folders to pull in, and you can add settings that are specific to that workspace without changing your global settings

Comment (thrile) [reply .vscode/cirkopt.code-workspace:8]: It won't need to be changed often, just because we are modifying the workflow
